core/src/main/java/org/mule/runtime/core/privileged/util/AttributeEvaluator.java

```diff
@@ -121,12 +90,12 @@ private boolean isParseExpression(String attributeValue) {
     return remainingString.contains(DEFAULT_EXPRESSION_POSTFIX);
   }
 
-  public <T> TypedValue<T> resolveTypedValueFromContext(BindingContext context) {
-    return expressionContextResolver.apply(context);
+  public <T> TypedValue<T> resolveTypedValue(ExpressionManagerSession session) {
```

Comment:
this class is API for privileged plugins, we can't remove the old method

Reply:
whoops